target/classes/application.properties

```diff
@@ -0,0 +1,11 @@
+#spring.datasource.url=
```

Comment:
Please add /target dir to .gitignore.
"target" dir is generated by maven when you "mvn clean install", we don't need this in git repository

Reply:
Added "target" dir to .gitignore.